### PR TITLE
Request to merge Feature final

### PR DESCRIPTION
**Pull request by Kingstalux**
- For this final milestone, I added some styles to improve the look and feel of the application.
- The list is also arranged in decending order.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
...
-
+>I am setting up a JavaScript project for the Leaderboard list app, using Webpack and ES6 features, notably modules. I have developed a first working version of the app following a wireframe, but with minimum styles just focusing on the functionality. Next, I've implemented the API functionality. A user can post their favorite Game of Thrones character with their score, which should be less than 1000. The data is then sent to the API and when the user clicks on the refresh button, the data is gotten from the API and displayed on the page. On refreshing the page, the correct list is gotten from the API and displayed. For the final milestone, I added some styles to improve the look and feel of the application.
 
 ## Built With
```
Comment (rloterh) [inline README.md:8]: - [x] The demo link on your ReadMe is **not updated** and therefore invalid. As this is the final touch to this project and may potentially feature on your portfolio, you are therefore **required** to have professional ReadMe for this project. Kindly host this project of GitHub Pages, Netlify, or the like, and provide the appropriate demo link on your ReadMe. Also, change the title _Webpack_ to **Live Demo** or similar.

**Your Demo Link:**
image

File: src/index.js [hunk 2]
```diff
@@ -64,3 +65,10 @@ form.addEventListener('submit', (e) => {
 document.getElementById('refresh').addEventListener('click', () => {
   getData();
 });
+
+// const getScores = async (cont) => {
+//   const response = await fetch(scoresUrl);
+//   const jsonData = await response.json();
+//   const sortArr = jsonData.result.sort((a, b) => b.score - a.score);
+//   frameCreation(sortArr, cont);
+// };
```
Comment (rloterh) [inline src/index.js]: - [x] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

**Review by rloterh — APPROVED** [review]
checked

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Kingsley, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- App is running as intended
- Nice UI design
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Kingstalux) [reply README.md:8]: Deploying the project was not part of the requirements. The link I provided is the local host link. If we were told to deployed I would have done so and provided the link.

File: src/index.js (hunk 2 above)
Comment (Kingstalux) [reply src/index.js]: okay I'll remove that.

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md:8]: Hi @Kingstalux 
You may simply remove the link. But just make sure that your ReadMe is clean, as maintaining a professional ReadMe is a requirement

Comment (Kingstalux) [reply README.md:8]: The link is used after you've run `npm start`, so I do not see how it is making my ReadMe unclean.

**Review by dasileker — APPROVED** [review]
### **Hi @Kingstalux,** 

### **Congratulations! 🎉  Project Approved ✅** 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 

Approved

 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**